### DISCUSSION TITLE
popovers sometimes are stuck - hovering over user triggers popover, but sometimes it will not go away until i reload page

### DISCUSSION THREAD

**Discussion by wiejakp** [community/community#45393] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

seems like there's an issue if I hover over link or a thing that triggers popover, sometimes they never close unless i hover on another thing that triggers another popover.

I feel like it happens when popover script is being called before the page fully render. only after i refresh the page it will be fixed and popover will work as should.

i use firefox 108.0.2 on macos

**Comment by gravyboat** [comment]
I'm running in to this as well on both MacOS and Windows on Firefox 109.0. It's most noticeable (and annoying) when looking at pull requests or issues. I can't duplicate this issue in Edge or Chrome currently.

The potential cause pointed out by @wiejakp seems to be accurate or at least point towards the root cause of the problem. Examples:

No issue:
* Go directly to https://github.com/streamlink/streamlink/pull/5044

Issue occurs every time:
* Go to https://github.com/streamlink/streamlink/pulls
* Click on the first PR in the list which will take you to pull 5044 from above (any PR will cause the issue), the issue is present on hover over cards
* Refresh the page (you should now be on pull/5044) and the issue vanishes

**Comment by Raboo** [comment]
Seems like github is aware and working on it.
https://github.com/orgs/community/discussions/44630

**Reply by gravyboat** [reply]
Nice find @Raboo, didn't realize this is something that belonged in general so I didn't look there, thanks! 👍
